addons/mrp/views/mrp_production_views.xml

```diff
@@ -96,13 +96,15 @@
                             <field name="product_id" attrs="{'readonly': [('state', '!=', 'draft')]}" domain="[('bom_ids', '!=', False), ('bom_ids.active', '=', True), ('bom_ids.type', '=', 'normal')]"/>
                             <field name="product_tmpl_id" invisible="1"/>
                             <label for="product_qty"/>
-                            <div class="o_row">
-                                <field name="product_qty" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
-                                <field name="product_uom_id" options="{'no_open':True,'no_create':True}" force_save="1" groups="uom.group_uom" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
-                                <button type="action"
-                                    name="%(mrp.action_change_production_qty)d"
-                                    context="{'default_mo_id': id}"
-                                    string="Update" class="oe_link" attrs="{'invisible': ['|', ('state', 'in', ('draft', 'done','cancel')), ('id', '=', False)]}"/>
+                            <div class="o_row no-gutters d-flex">
+                                <div class="col">
+                                    <field name="product_qty" attrs="{'readonly': [('state', '!=', 'draft')]}"/>
```

Comment:
i don't understand this commit, can you explain?

Reply:
here, remove negative margin from default. and put it in one column.